apps/dialtone-documentation/docs/components/badge.md

```diff
@@ -259,12 +289,38 @@ Decorative badges label and classify items for quick recognition.
   </dt-stack>
 </code-well-header>
 
-```html
+<code-example-tabs
+htmlCode='
 <span class="d-badge d-badge--decorate-{$color}">
   <span class="d-badge__decorative"></span>
   <span class="d-badge__label">Label</span>
 </span>
-```
+'
```

Comment:
The purple on the above example is out of alignment. I know it wasn't your change that caused this but we should probably fix it while we're in here.

![Screenshot 2024-03-04 at 5 00 52 PM](https://github.com/dialpad/dialtone/assets/64808812/3d9a9775-31bf-41a7-adc6-ee3464336ca1)

Reply:
ofc, I didn't see it., thanks